lib/path.js

```diff
@@ -22,17 +22,7 @@
 'use strict';
 
 const errors = require('internal/errors');
-const {
-  CHAR_UPPERCASE_A,
-  CHAR_LOWERCASE_A,
-  CHAR_UPPERCASE_Z,
-  CHAR_LOWERCASE_Z,
-  CHAR_DOT,
-  CHAR_FORWARD_SLASH,
-  CHAR_BACKWARD_SLASH,
-  CHAR_COLON,
-  CHAR_QUESTION_MARK,
-} = require('internal/constants');
+const { ASCII } = require('internal/constants');
```

Comment:
This results in a potential extra lookup and unnecessary churn. It can be simplified by changing the original line 35 to `} = require('internal/constants').ASCII;`

Due to the name change of the constants it would still be necessary to change the entries here but the extra lookup would be gone.